packages/app/pubspec_overrides.yaml

```diff
@@ -20,5 +20,10 @@ dependency_overrides:
     path: ../neon/neon_notifications
   nextcloud:
     path: ../nextcloud
+  package_info_plus:
+    git:
+      url: https://github.com/fluttercommunity/plus_plugins
+      ref: baa336fe79c29411542fc343d332846bcd7752c1
+      path: packages/package_info_plus/package_info_plus
   sort_box:
     path: ../sort_box
```

Comment:
When I run `melos clean && melos bs` sort_box is before the new override

Reply:
melos clean was not deleting `packages/app/pubspec_overrides.yaml` for me. After a manual rm and a `melos bs` it works.
Strange